### PR TITLE
Get prepared for 0.6.2

### PR DESCRIPTION
**Pull request by elemoine**
I'd like to release 0.6.2, with the [support of WKBElement objects as bind values](https://github.com/geoalchemy/geoalchemy2/pull/221), which I consider as a bug fix rather than a new feature.

### CODE REVIEW

**Comment by adrien-berchet** [comment]
I just added a minor commit to update my account name in the contributors list since it was deprecated.

File: doc/conf.py [hunk 1]
```diff
@@ -49,9 +49,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '0.6.1'
+version = '0.6.2'
```
Comment (sbrunner) [inline doc/conf.py]: The version pattern don't correspond to the comment ...

Comment (elemoine) [reply doc/conf.py]: It's ok I think. `version` is the actual the version number, while `release` is the version number + the alpha/beta/rc tag. In our case `version` and `release` are the same, and we use three-digit version numbers.

**Comment by elemoine** [comment]
> I just added a minor commit to update my account name in the contributors list since it was deprecated.

Which I just amended to remove the final dot in the commit message :)

**Comment by adrien-berchet** [comment]
> Which I just amended to remove the final dot in the commit message :)

Ups, sorry and thanks :innocent:

**Comment by elemoine** [comment]
0.6.2 is released on pypi: https://pypi.org/project/GeoAlchemy2/0.6.2/

File: doc/conf.py (hunk 1 above)
Comment (sbrunner) [reply doc/conf.py]: As you want but it doesn't respect the comment :-)

Comment (elemoine) [reply doc/conf.py]: Yes, that's a bit error-prone. I'll change it.

Comment (sbrunner) [reply doc/conf.py]: :-)

**Pull request merged**